packages/client/src/Compose/RecipientsInput.tsx

```diff
@@ -67,46 +101,130 @@ function reducer(state: State, action: Action): State {
+    return suggestion.name
+      ? `${suggestion.name} <${suggestion.mailbox}@${suggestion.host}>`
+      : `${suggestion.mailbox}@${suggestion.host}`
+  }
+
+  const handleSuggestionsFetchRequested = () => {
+    return suggestions
+  }
+
+  const handleSuggestionsClearRequested = () => {
+    suggestions = []
+  }
+
+  const onSuggestionSelected = () => {
+    dispatch({ type: "parseAddresses" })
+  }
+
+  function renderInputComponent(
+    inputProps: Autosuggest.InputProps<typeof suggestions[number]>
+  ) {
+    return (
+      <TextField
+        {...(inputProps as any)}
+        {...rest}
+        fullWidth
+        InputProps={{
+          startAdornment: (
+            <InputAdornment position="start">
+              {recipients.map(recipient => (
+                <Chip
+                  key={email(recipient)}
+                  avatar={
+                    <Avatar
+                      address={{
+                        name: recipient.name,
+                        mailbox: recipient.local,
+                        host: recipient.domain
+                      }}
+                    />
+                  }
+                  tabIndex={-1}
+                  label={display(recipient)}
+                  className={classes.chip}
+                  onDelete={() => dispatch({ type: "remove", recipient })}
+                />
+              ))}
+            </InputAdornment>
+          )
+        }}
+        InputLabelProps={{
+          shrink: recipients.length > 0 || inputValue.length > 0 || focused
+        }}
+      />
+    )
+  }
+
+  const autosuggestProps = {
+    renderInputComponent,
+    suggestions: skip ? [] : suggestions,
+    onSuggestionsFetchRequested: handleSuggestionsFetchRequested,
+    onSuggestionsClearRequested: handleSuggestionsClearRequested,
+    onSuggestionSelected,
+    getSuggestionValue,
+    renderSuggestion
+  }
+
   React.useEffect(() => {
     onRecipients(recipients)
   }, [onRecipients, recipients])
 
   return (
-    <TextField
-      {...rest}
-      InputProps={{
-        startAdornment: (
-          <InputAdornment position="start">
-            {recipients.map(recipient => (
-              <Chip
-                key={email(recipient)}
-                avatar={
-                  <Avatar
-                    address={{
-                      name: recipient.name,
-                      mailbox: recipient.local,
-                      host: recipient.domain
-                    }}
-                  />
-                }
-                tabIndex={-1}
-                label={display(recipient)}
-                className={classes.chip}
-                onDelete={() => dispatch({ type: "remove", recipient })}
-              />
-            ))}
-          </InputAdornment>
-        ),
-        onChange: event =>
-          dispatch({ type: "inputChange", value: event.target.value }),
+    <Autosuggest
+      {...autosuggestProps}
+      inputProps={{
+        id: "react-autosuggest-simple",
```

Comment:
I think that this `id` is not necessary. The documentation says, "Use it only if you have multiple Autosuggest components on a page."